src/Components/Counter.jsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
-import Display from './Display';
+import React, {useState} from 'react';
+import  "./DisplayCss.css";
+import Display from "../Components/Display"
 import ButtonPad from './ButtonPad';
 
 
-function Counter() {
+const Counter=()=> {
+    const [count, setCount] = useState(0);
+    const [list, setList]= useState([]);
+    const handleCount = (handle)=>{
+        if(handle == 0){
+            setCount(0)
+            setList([0])
+            return 
+        }
+        setCount(count+handle);
+        let newItem = count+handle;
+        let newList = list;
+        newList.push(newItem);
+        setList(newList);
+        
+    }
     return (
         <div className="counter">
-            <Display/>
-            <ButtonPad/>
+            <Display displayCount={count} listCount={list} />
+            <ButtonPad 
+            onIncremente={(e)=>{
+                e.nativeEvent.stopImmediatePropagation()
+                handleCount(1)
+            }}
+            onIncrementeTwo={()=>handleCount(2)}
+            onIncrementeFive={()=>handleCount(5)}
+            onDecremente={()=>handleCount(-1)}
```

Comment:
My instinct here is that these increment / decrement functions can be combined into a single function.  As @craigstockton likes to phrase it, "encapsulate what changes".